promql/parser/lex_test.go

```diff
@@ -16,6 +16,7 @@ package parser
 import (
 	"testing"
 
+	"github.com/prometheus/prometheus/promql/parser/position_range"
```

Comment:
This has to go into a separate block below, separated by a blank line from `"github.com/stretchr/testify/require"`.